[PL-46862] PL 1.24.7 Release Notes (#5276)
harness/developer-hub: release-notes/platform.md

## February 2024

### Version 1.24.4 <!--  February 12, 2024 -->

#### New features and enhancements

> - Users can now enable file logging for supported services through override in On-prem installations. (PL-44211)

Reviewer: ```suggestion
- You can now enable file logging for supported services through override in Harness Self-Managed Enterprise Edition (On-prem) installations. (PL-44211)
```

>    To enable file logging add the following to your override in the global section:

Reviewer: ```suggestion
   To enable file logging, add the following to your override in the global section:
```

>    ```

Reviewer: ```suggestion
   ```yaml
```

>    global:

Reviewer: Recommend fixing the YAML spacing. https://harness.atlassian.net/browse/PL-44211?focusedCommentId=666095

   fileLogging:
   enabled: true
   logFilename: /opt/harness/logs/pod.log #provide log filename
   maxFileSize: 50MB #max single file size, for log archiving
   maxBackupFileCount: 10 #max count of files
   totalFileSizeCap: 1GB

   ```

> - Support added for encrypted assertions in the SAML response. (PL-43353)

Reviewer: ```suggestion
- Added support for encrypted assertions in the SAML response. (PL-43353)
```

#### Fixed issues

> - Users were unable to change the authentication mechanism of a secret manager from a non-password based to a password based mechanism due to a bug in the secret manager update API. (PL-46657)

Reviewer: ```suggestion
- Users weren't able to change the authentication mechanism of a secret manager from a non-password-based to a password-based mechanism due to a bug in the secret manager update API. (PL-46657)
```

>    This issue has been fixed, enabling the modification of the authentication mechanism for secret managers to a password based mechanism in this release.

Reviewer: ```suggestion
   This issue has been fixed, enabling the modification of the authentication mechanism for secret managers to a password-based mechanism.
```
